src/OrleansAWSUtils/Storage/SQSStorage.cs

```diff
@@ -0,0 +1,233 @@
+﻿using Amazon.Runtime;
+using Amazon.SQS;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.SQS.Model;
+using Orleans;
+using SQSMessage = Amazon.SQS.Model.Message;
+
+namespace OrleansAWSUtils.Storage
+{
+    /// <summary>
+    /// Wrapper/Helper class around AWS SQS queue service
+    /// </summary>
+    public class SQSStorage
+    {
+        /// <summary>
+        /// Maximum number of messages allowed by SQS to peak per request
+        /// </summary>
+        public const int MAX_NUMBER_OF_MESSAGE_TO_PEAK = 10;
```

Comment:
lol Yeah... I'm not going comment on that :P Believe me... There are worse things than that ;)